exercises/practice/palindrome-products/.approaches/factors-first/content.md

```diff
@@ -0,0 +1,101 @@
+# Factors first
+
```

Comment:
We usually list the entirety of the solution in the beginning of the file.

Reply:
In that case I'll have to develop a full solution :)